tests/readwrite/test_xgi_data.py

```diff
@@ -48,10 +48,29 @@ def test_load_xgi_data(capfd):
     assert "email-enron" in out
     assert "congress-bills" in out
 
+    # test collection
+    collection = load_xgi_data("hyperbard")
+    assert len(collection) == 37
```

Comment:
this is the number of datasets in the collection right?

Reply:
Correct!